packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -363,10 +371,40 @@ class FileCollection extends DocumentCollection {
     if (!href) throw new Error('No related link in server response')
     return this.stackClient.fullpath(href)
   }
-
-  async download(file) {
-    const href = await this.getDownloadLinkById(file._id)
-    forceFileDownload(`${href}?Dl=1`, file.name)
+  /**
+   * Download a file or a specific version of the file
+   *
+   * Passing a filename to forceFileDownload is not enought
+   * for a few browsers since the stack's response header will
+   * not contain that name. Passing the filename to getDownloadLinkBy*
+   * will ask the stack to return this filename in its content-disposition
+   * header response
+   *
+   * @param {object} file io.cozy.files object
+   * @param {string} versionId Id of the io.cozy.files.version
+   * @param {string} filename The name you want for the downloaded file
+   *                            (by default the same as the file)
+   */
+  async download(file, versionId = null, filename = '') {
```

Comment:
Is there a reason to use an empty string instead of undefined ?

Reply:
Not needed anymore. Thanks